src/Components/DonationFormModal.js

```diff
@@ -164,16 +278,13 @@ const DonationFormModal = () => {
             <input
               data-testid="image-upload"
               type="file"
-              value={formState.imageUpload}
+              accept="image/png, image/jpeg"
+              value={formState.imageName}
               id="imageUpload"
-              onChange={(e) =>
-                setFormState({
-                  ...formState,
-                  imageUpload: e.target.value
-                })
-              }
+              onChange={handlePic}
```

Comment:
This was a great learning to upload an image and saving it through cloudinary and using the response grab the URL of the image!!